include/internal/sockets.h

```diff
@@ -208,10 +208,6 @@ typedef size_t socklen_t;        /* Currently appears to be missing on VMS */
 # endif
 
 /* also in apps/include/apps.h */
-# if defined(OPENSSL_SYS_WIN32) || defined(OPENSSL_SYS_WINCE)
-#  define openssl_fdset(a, b) FD_SET((unsigned int)(a), b)
```

Comment:
Sorry, I missed this previously. I think you need to keep this for `OPENSSL_SYS_WIN32` and probably add parentheses for the sake of hygiene:

```C
# if defined(OPENSSL_SYS_WIN32)
#  define openssl_fdset(a, b) FD_SET((unsigned int)(a), (b))
# else
#  define openssl_fdset(a, b) FD_SET((a), (b))
# endif
```

Reply:
fixup pushed.  you have nothing to apologize for, I should have done a better job :)